### ISSUE TITLE
Disable logging?

### ISSUE THREAD

**Issue by davinderkumar** [ShivamArora/meteorify#20]
How to disable console logs?

**Comment by ShivamArora** [comment]
Hi @davinderkumar ,

Can you please give a brief about which logs are you talking about?

**Comment by davinderkumar** [comment]
Console logs

**Comment by ShivamArora** [comment]
Can you post a screenshot of the same?

**Comment by davinderkumar** [comment]
you have many prints in your code
<img width="831" alt="screenshot" src="https://user-images.githubusercontent.com/5299516/112240692-ce474300-8c6e-11eb-8e3e-255ee678eb6e.png">

**Comment by ShivamArora** [comment]
These logs are coming from an underlying package `ddp` that meteorify is using. 
I'll just push up some changes in a separate branch that will help you out in disabling the logs.

But I'll need your help in order to test those changes as I don't have access to a meteor server to connect to right now.

I hope you'll be available for that.

**Comment by ShivamArora** [comment]
Hi @davinderkumar ,

Can you please test the changes in the [feature/disable-ddp-logging](https://github.com/ShivamArora/meteorify/tree/feature/disable-ddp-logging) branch?

All you have to do is to switch the dependency resolution to point to this branch using:

```
dependencies:
  meteorify:
    git:
      url: git@github.com:ShivamArora/meteorify.git
      ref: feature/disable-ddp-logging
```
and 

make your connection by providing the `enableLogs` parameter like this:
```
var status = await Meteor.connect('ws://example.meteor.com/websocket', enableLogs: false);
```

Let me know if that works for you and I'll push an update accordingly.

**Comment by davinderkumar** [comment]
error 
<img width="970" alt="pubspec_yaml_—_mobile" src="https://user-images.githubusercontent.com/5299516/114751033-48e91700-9d72-11eb-8265-7f20b96eff7e.png">

**Comment by davinderkumar** [comment]
Also, can you publish an updated version for latest flutter.

**Comment by davinderkumar** [comment]
Logging is still there

<img width="627" alt="logs" src="https://user-images.githubusercontent.com/5299516/114751937-679bdd80-9d73-11eb-909d-6dddd004dec0.png">

**Comment by ShivamArora** [comment]
> Also, can you publish an updated version for latest flutter.

The package doesn't depend on flutter sdk. It only uses Dart under the hood.

Is there something that I misunderstood?

**Comment by ShivamArora** [comment]
> Logging is still there
> 
> <img alt="logs" width="627" src="https://user-images.githubusercontent.com/5299516/114751937-679bdd80-9d73-11eb-909d-6dddd004dec0.png">

Did you use `enableLogs: false`?

**Comment by davinderkumar** [comment]
Yes
> Did you use enableLogs: false?

DDP is dependent on some older version libraries.   I am getting this error
meteorify >=1.0.3 depends on ddp ^0.0.13 which depends on tuple ^1.0.2, meteorify >=1.0.3 requires tuple ^1.0.2.
> The package doesn't depend on flutter sdk. It only uses Dart under the hood.

**Comment by ShivamArora** [comment]
> DDP is dependent on some older version libraries. I am getting this error
> meteorify >=1.0.3 depends on ddp ^0.0.13 which depends on tuple ^1.0.2, meteorify >=1.0.3 requires tuple ^1.0.2.
> 
> >

I think the error message is a bit incomplete or wrong regarding the required version.
But I get the point that you might have started using the latest flutter sdk which has support for Dart Null Safety which requires tuple ^2.0.0.

Is there any other package as well that you see that needs to be upgraded?

**Comment by davinderkumar** [comment]
Yeah.
And because tuple >=1.0.2 <2.0.0-nullsafety.0 depends on quiver >=0.22.0 <3.0.0, meteorify >=1.0.3 requires quiver >=0.22.0 <3.0.0.

> I think the error message is a bit incomplete or wrong regarding the required version.

**Comment by ShivamArora** [comment]
@davinderkumar 
Seems like the DDP dependency that I use [https://github.com/haoguo/ddp](https://github.com/haoguo/ddp) is no longer under maintenance.
That's where the `tuple` and `quiver` dependencies are being used and since they've upgraded the major version it might bring some breaking changes to DDP.

I'll try if I can do something about it in my free time. But it would surely take a large chunk of time and might have to publish an up-to-date version of DDP myself.

For now, I'll create an issue on the DDP repository and see if I get a response.

**Comment by davinderkumar** [comment]
@ShivamArora 
Can you check this repo https://github.com/wendellrocha/enhanced_ddp

**Comment by ShivamArora** [comment]
@davinderkumar Just checked. But even this fork is using the older version of `tuple`.

And couldn't find any info on what has been enhanced/extended from the original repo.

It shows the fork is 78 commits ahead of the original repo, but the README is still the same and doesn't provide any info on what has been changed/added.

**Comment by davinderkumar** [comment]
yeah. most of it is just addition of enhanced word, using web_socket_channel, and remove listener.

**Comment by davinderkumar** [comment]
https://github.com/ShivamArora/meteorify/blob/feature/disable-ddp-logging/lib/src/meteor/subscribed_collection.dart
file contains lot of prints
